Source/FolioReaderConfig.swift

```diff
@@ -37,12 +39,54 @@ public enum FolioReaderScrollDirection: Int {
     }
 }
 
+// MARK: - ClassBasedOnCLickListener
+
+/**
+A `ClassBasedOnCLickListener` takes a closure which is performed if a given html `class` is clicked. The closure will reveice the content of the specified parameter.
+
+Eg. A ClassBasedOnCLickListener with the className "quote" and parameterName "id" with the given epub html content "<section class="quote" id="12345">" would call the given closure on a click on this section with the String "12345" as parameter.
+
+*/
+public struct ClassBasedOnCLickListener {
```

Comment:
We could simplify this name, it is a little confused for me, what do you think?

Also there is a typo `ClassBasedOnCLickListener`